scRNA-seq-advanced/01-read_filter_normalize_scRNA.Rmd

```diff
@@ -431,11 +432,11 @@ The default number of components is 50, which we will stick with, but let's ente
 
 ```{r runPCA, live=TRUE}
 # calculate and save PCA results
-normalized_sce <- scater::runPCA(normalized_sce,
-                                 # how many components to keep
-                                 ncomponents = 50,
-                                 # use only the variable genes we chose
-                                 subset_row = hv_genes)
+normalized_sce <- scater::runPCA(
+  normalized_sce,
+  ncomponents = 50, # how many components to keep
+  subset_row = hv_genes # use only the variable genes we chose
```

Comment:
do you want to keep this comment? up to you!

Reply:
I think it is good for the rendered nb.